proto/cosmos/crisis/v1beta1/tx.proto

```diff
@@ -7,9 +7,10 @@ import "gogoproto/gogo.proto";
 
 // MsgVerifyInvariant represents a message to verify a particular invariance.
 message MsgVerifyInvariant {
-  option (gogoproto.equal) = true;
+  option (gogoproto.equal)            = false;
+  option (gogoproto.goproto_getters)  = false;
 
-  bytes  sender = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+  string  sender = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
```

Comment:
I have a question here: `sender` is String, `AccAddress` is []byte. So I see 3 solutions:

1. Keep it as-is. Does it work? How does casting string to bytes work?
2. Don't use `casttype`. Maybe needs a lot of refactoring, because we'll basically drop AccAddress altogether.
3. Use `type AccAddress string` instead of `type AccAddress []byte`. I would say this is the path that allows the least refactor.

I'm seeing 1&2 used in this PR. @anilCSE Have you tried 3?

Reply:
`casttype` won't work. `customtype` might. 3) would be a larger SDK wise refactor and is probably not desirable

Reply:
Yes, I started initially with (3) and it looked like a bigger refactoring. So switch to the current approach. Yes, I am removing all  `casttype` declarations on addresses, it's still in progress